features/release/breadcrumbs.feature

```diff
@@ -50,12 +49,12 @@ Feature: Attaching a series of notable events leading up to errors
     And I invoke "notify_error"
     And I wait to receive an error
     And I discard the oldest error
+    And I invoke "notify_error_on_foreground"
     # Now we know that the backgrounding will occur at an appropriate time
     And I switch to the web browser for 2 seconds
     # Give iOS sufficient time to raise the notification for foregrounding the app
     And I make the test fixture wait for 1 second
```

Comment:
We don't need this line and comment now (the fixture may fail to get the command it tries to send anyway).